### ISSUE TITLE
[feature]:  Time tracking

### ISSUE THREAD

**Issue by jaavid** [makeplane/plane#1184]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Summary

is there any integration for time tracking?
like jira tempo 

### Why should this be worked on?

to make the pmo teams happy !

**Comment by vishalpatelvc** [comment]
We are working on it.... Probably will raise PR very soon

**Comment by rhea0110** [comment]
Hey @jaavid , thank you for submitting this feature request. If you are referring to time tracking for issues, I'm pleased to inform you that we have already begun working on it and expect to release it sometime soon. We will keep you updated on the progress.

Thank you!

**Comment by amishsthapit** [comment]
I am also waiting for the time tracking feature and associated reports for it to be viable for my organization.

**Comment by mmospanenko** [comment]
Will be great to have Clockify integration, Estimated Time, Spent Time, Remaining Hours, Progress. So that will allow us to know if everything goes as expected.

How it works

As a developer I prepare plan (ToDo List) with steps and each step I estimate in hours

- [ ] do one: 1
- [ ] do two: 2

so Estimated Time is 3 and when I will spend more one hour (from Clockify data) - I will know that I have to change ETA right now (adding more hours, etc). And I always know that I will not fail Due Date, it is really important

What functionality is required
1. ToDo Lists (with crossing)
2. Original Estimate, Remaining Hours (like Jira has) fields
2. WorkLogs (Clockify or api integrations) to set Spent Time
3. Some calculated fields - like Progress %

**Comment by leikoilja** [comment]
Can't wait for this feature🎉🥲💪🏼
@rhea0110 , are there any updates?

**Comment by oberonlai** [comment]
Any time tracking updates? This is the most important factor when I choose the project management software.

**Comment by cgallay** [comment]
I totally agree with @oberonlai I am currently assessing multiple project management software. Plane looks nice and promising, but the lack of time tracking capability will probably make us chose another solution as of now.

**Comment by sharad-paghadal** [comment]
Any update on time tracking support? I'm eagerly waiting for it, if provided even beta or alpha version, that will be also fine.

Thanks.

**Comment by maxdd** [comment]
Any update?

**Comment by sharad-paghadal** [comment]
No they are still doing research on best approach for time management. That is what I had been told by someone on discord channel.

So I think it's likely not coming at least for 4 months.

**Comment by leopathu** [comment]
It would be better to have its own time-tracking features instead of integration with third-party services.

**Comment by FallenChromium** [comment]
@leopathu As far as I understand there's no way to deeply integrate those as plugins into the plane.so since SDK isn't available yet, or am I having outdated information?

**Comment by Winter** [comment]
Looks like an amazing project. The lack of time tracking features really renders it unusable for a lot of organisations though. Will subscribe to the issue and hopefully we get an update in the future. Key features we're after:

1. Aggregate time management. IE: Ability to say "spent 15 minutes here", "another 20 minutes there", while also adding a comment for what was done
2. Granular levels of reports for what work was completed and when

**Comment by mmospanenko** [comment]
This could be covered with https://github.com/makeplane/plane/issues/2865

**Comment by luis-guimaraes-exoawk** [comment]
> Looks like an amazing project. The lack of time tracking features really renders it unusable for a lot of organisations though. Will subscribe to the issue and hopefully we get an update in the future. Key features we're after:
> 
>     1. Aggregate time management. IE: Ability to say "spent 15 minutes here", "another 20 minutes there", while also adding a comment for what was done
> 
>     2. Granular levels of reports for what work was completed and when

This would be our exact use case, subscribing to the issue for updates

**Comment by petrkr** [comment]
Just looking for alternative to Jira, as they canceled my account, because I not logged it for half year...

So looking for some on-premise. This project looks very nice, but I need tracks time too. Mentioned custom field is not usable, as it does not support aggregations. And if I have to use Libreoffice Calc to make it, then it make pointless to using tracking system at all.

If some donation will help in order make this feature on higher priority. I can donate something. Still better than pay to atlassian for their overpriced products.

**Comment by sharad-paghadal** [comment]
I would say - apply for clickup startup and get 1 year of free usage from them. by that time plane will have more features including time tracking - then we can safely migrate.

**Comment by erickhaendel** [comment]
release date? Extremely important functionality for workload management.

**Comment by Dan6erbond** [comment]
Any updates on this?

**Comment by domeradzki** [comment]
Any news with this feature?

**Comment by joao-intelliway** [comment]
The absence of this feature is holding my company back on adopting Plane as main project management tool

**Comment by leopathu** [comment]
Yes, For me too

**Comment by hadifarnoud** [comment]
@rhea0110 any update on this?

**Comment by theparthacus** [comment]
quick update. Time Tracking's out on One and Pro. see [this](https://plane.so/pricing#project-work-management) for a full comparison of the section that houses Time Tracking and scroll through the page to see how much you save.

**Comment by hadifarnoud** [comment]
There is no free trial on cloud version even? how can I see how it works

**Comment by theparthacus** [comment]
there will be a Pro trial on the Cloud when Pro launches or a little before that.

**Comment by petrkr** [comment]
And will be some version for someone like me? Who just want track time on own free and open source projects?

In business mostly company already have something paid.

I was looking for free replacement of Jira, which stop offer on-premise solution and offers only cloud. But not your hardj-drive, not your data.

**Comment by mmospanenko** [comment]
seems we should move away from Plane, disappointed with their politics and directions. So small feature and only for "corporations", very strange

**Comment by leopathu** [comment]
I feel the same. very disappointed with this.

**Comment by sharad-paghadal** [comment]
This is only because there was a lot of buzz about this feature, and they saw an opportunity to make some money from it. However, they don’t seem to understand that people were genuinely interested in using the platform because of its open-source nature. If I had to pay to track my time, I wouldn’t use it at all. They missed out on the chance to onboard large companies to use the platform within their organizations. @theparthacus

**Comment by hadifarnoud** [comment]
I understand why they're doing this though. This is a business and they have to make money. I don't mind paying if I had a big enough company to justify the price.

It is harder to sell support plans with large companies. This is why they're doing this. Fair enough

**Comment by hadifarnoud** [comment]
there are a lot of examples of fully open source software with paid hosted versions, like Sentry for example. All I'm saying is we don't know how things are in Plane. I would love to have everything without paying anything, and just pay for support when we grow bigger.

**Comment by mmospanenko** [comment]
@sharad-paghadal absolutely. We use Notion for more mature organizations where we can pay and get more solid product (moved from Jira after ~10 years). Plane is like "rising star", testing yet, but current behavior will kill any opportunities for them.

**Comment by Beyshka** [comment]
It's really a pity that you can't plan by time, but only by date (number). It's very difficult to plan the day, since all the tasks are pouring into a heap...

**Comment by sTiKyt** [comment]
The only tool that does time tracking right is Bitrix24, but Bitrix24 is terrible in every other way

I'm surprised to see most other project management software is incapable of just adding a "play/pouse" button next to each task so people can hit "Play" when they start working and "Pause" or "Stop" when they are done


Manual logging relies on people remembering to add time/to start tracking elsewhere beforehand, it's not reliable

**Comment by leopathu** [comment]
With the help of AI. I am building my own Project management tool with time tracking features for my organisation  https://github.com/leopathu/quill 

Most of the things are ready and working. Still, there is some more work and improvements.

**Comment by hadifarnoud** [comment]
> The only tool that does time tracking right is Bitrix24, but Bitrix24 is terrible in every other way
> 
> I'm surprised to see most other project management software is incapable of just adding a "play/pouse" button next to each task so people can hit "Play" when they start working and "Pause" or "Stop" when they are done
> 
> Manual logging relies on people remembering to add time/to start tracking elsewhere beforehand, it's not reliable

I'm currently looking for a simple open source kanban project management software that I can add time tracking feature to it myself

**Comment by jontorrado** [comment]
As far as I know, Plane already have a Time Tracking solution, but not in the free plan. It starts from the Pro plan. We've tried that solution and it's simply not what we want, so we are currently using both Clockify or solidtime (Open Source) depending on the project.

Hope this helps.

Cheers.

**Comment by vihar** [comment]
Closing this one — time tracking is part of our Pro and Business plans now: https://docs.plane.so/core-concepts/issues/time-tracking

Thanks for the patience here.

**Comment by leopathu** [comment]
Don't say it is a open source

**Comment by sTiKyt** [comment]
> Closing this one — time tracking is part of our Pro and Business plans now: https://docs.plane.so/core-concepts/issues/time-tracking
> 
> Thanks for the patience here.

So this project is not open-source, it's open-core...
